### DISCUSSION TITLE
Dependabot link in README is private

### DISCUSSION THREAD

**Discussion by timvisee** [eza-community/eza#226] [Q&A, Howto, Help] [answered]
The README shows security fixes as feature, it links to a dependabot security page.

> Several security fixes (see [dependabot](https://github.com/eza-community/eza/security/dependabot?q=is%3Aclosed))

I cannot access this link however. I believe this page is only available to repository administrators:

![image](https://github.com/eza-community/eza/assets/856222/003c24ea-816d-4763-b090-c9c85d5f2f15)

If this is true, I think it would be better to remove this link. It does not make sense to advertise with a feature that has a non functional link.

It is possible to link to [dependabot PRs](https://github.com/eza-community/eza/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+), but that is not the same.

**Comment by cafkafk** [comment, marked as answer]
I've removed the link, thanks for making me aware.
